plugins/outputs/wavefront/README.md

```diff
@@ -24,21 +24,20 @@ to use them.
 
 ```toml @sample.conf
 [[outputs.wavefront]]
-  ## Url for Wavefront API or Wavefront Proxy instance.
+  ## Url for Wavefront API or Wavefront proxy instance.
   url = "https://metrics.wavefront.com"
 
-  ## Authentication Token for Wavefront. Required if using Direct Ingestion. Not required if using a Wavefront Proxy.
-  #token = "DUMMY_TOKEN"
-
-  ## Maximum number of metrics to send per HTTP request. This value should be higher than the `metric_batch_size`. Default is 10,000. Values higher than 40,000 are not recommended.
-  # http_maximum_batch_size = 10000
-
-  ## Deprecated. DNS name of the Wavefront server or Wavefront Proxy. Use the `url` field instead.
+  ## Deprecated. DNS name of the Wavefront server or Wavefront proxy.
```

Comment:
Should we remove the deprecated options altogether? I mean this is an example config....

Reply:
I'm good with that too

Reply:
I love the idea of removing it from the example!

Reply:
Because I'm removing host/port from the example, I want to add back a warning log in the plugin. I've had some Wavefront users complain that they get the deprecation warning about host/port, but they don't feel confident rewriting their host/port as a URL. So I want to include a log statement that tells them what URL we're using for them, so that (hopefully) they can see that log line and the deprecation warning and just update their `telegraf.conf` accordingly.